lib/boxen/preflight/creds.rb

```diff
@@ -105,4 +110,24 @@ def fetch_from_env(thing)
     warn "Oh, looks like you've provided your #{thing} as environmental variable..."
     found
   end
+
+  def fingeprint
```

Comment:
Yup :smile: - had that typo fixed up on my local branch but haven't pushed it up yet.